config/src/main/java/org/springframework/security/config/annotation/web/configurers/AuthorizeHttpRequestsConfigurer.java

```diff
@@ -70,8 +70,13 @@ public AuthorizationManagerRequestMatcherRegistry getRegistry() {
 		return this.registry;
 	}
 
+	@SuppressWarnings("unchecked")
 	@Override
 	public void configure(H http) {
+		ExpressionUrlAuthorizationConfigurer<?> configurer = http
```

Comment:
Can we move this to where we are building http? I'd prefer the configurers not need to know about each other.